com.unity.render-pipelines.universal/Runtime/Passes/DepthNormalOnlyPass.cs

```diff
@@ -14,9 +14,12 @@ public class DepthNormalOnlyPass : ScriptableRenderPass
         internal ShaderTagId shaderTagId { get; set; } = k_ShaderTagId;
 
         private RenderTargetHandle depthHandle { get; set; }
+        private RenderTargetHandle cameraDepthHandle { get; set; }
```

Comment:
camera depth can be accessed from within pass by calling renderingData.cameraData.renderer.cameraDepthTarget.

Reply:
Fixed